src/test/test-backwards-compatible.gd

```diff
@@ -65,6 +65,14 @@ func test_timestamp_created() -> void:
 	assert_true(history_ultra.timestamp.has("second"))
 
 
+func test_ultra_compare() -> void:
+	assert_true(PlayerData.scenario_history.scenario_names().has("ultra-normal"))
+	var history_ultra: RankResult = PlayerData.scenario_history.results("ultra-normal")[0]
+	
+	# new rank results include a 'compare' field so we know how to compare them
+	assert_eq(history_ultra.compare, "-seconds")
```

Comment:
Add a counterpart to this test to ensure 'sprint' records have the correct 'compare' value

Reply:
Added `test_compare_score` test.